docs/sources/tempo/configuration/_index.md

```diff
@@ -412,8 +412,8 @@ metrics_generator:
             # Enable traces_target_info metrics
             [enable_target_info: <bool> | default = false]
 
-            # Attribute Key to multiply span metrics
-            [span_multiplier_key: <string> | default = ""]
+            # Attribute Key to multiply metrics
+            [multiplier_key: <string> | default = ""]
```

Comment:
we can change this config name, but we need to mark it a breaking change. i don't see a particularly strong reason to change it. the behavior has changed some, but it still is a "multiplier for span metrics".